### ISSUE TITLE
Japanese Early Access font issues

### ISSUE THREAD

**Issue by davelab6** [google/fonts#529]
https://twitter.com/jutha3_21/status/806666758450913280 mentions a few issues (text via Google Translate)

> Well, as early as 2017 version of the Japanese free font summary has come out. The one I often use is Hanpei Ming Dynasty, M +, around a cutaway. Also, why is not there in this list, but Y.OzFont (http://yozvox.web.fc2.com) too.
>
> Since the poetry of http://tanakajutha.tumblr.com gives priority to Ming Dynasty, it is glad if you can install it and view it. Other places prioritize Yu Gothic (font - weight: 500px;) or Yu Mincho entered in the latest Windows and Mac.
> 
> Hannari Mincho tried the web font of Google Fonts + (https://googlefonts.github.io/japanese/), but in my parents' environment (Windows 10, Google Chrome), for example, only Hiragana was applied So abandon. There is also a disadvantage that capacity increases.

**Comment by laerm0** [comment]
So, to summarize, I think the issues are:

- [ ] add Y.OzFont
- [ ] add Ming Dynasty
- [ ] Hannari Mincho has issues with Chrome on Win 10

**Comment by HinTak** [comment]
Google's translate is a bit crap...  "around a cutaway" should be "outline font", and "was applied So abandon" should be "was suitable for use, so abandone the thought (i.e. forget about it)".

**Comment by davelab6** [comment]
Thanks for the correction hintak :) 

Micah, summary sounds good. 

Triaging the hannari problem is the next step here,

**Comment by laerm0** [comment]
Okay, so here's Chrome on Win 10:

![screen region 2017-09-22 at 16 02 53](https://user-images.githubusercontent.com/10780161/30762302-8f15c1c0-9faf-11e7-8ff0-7d62281abc9f.png)

And then Chrome on OS X:

![screen region 2017-09-22 at 16 03 02](https://user-images.githubusercontent.com/10780161/30762316-975e9ee2-9faf-11e7-8e63-d6c077ef9c7a.png)

I'm pretty sure those kana are identical, besides the usual Windows look of text a little darker than OS X. Here's the [jsbin](http://jsbin.com/nosohexoge/edit?html,output). Thoughts?

**Comment by HinTak** [comment]
Chrome on OS X is substituting the Kanji part with a typeface that's not mincho (more like gothic).

On my linux box, firefox 55.0.3 is not showing any the kanji (the first of the sample 3 glyphs) https://googlefonts.github.io/japanese/ at the bottom for hannari, nor Kokoro. Nico Moji and Nikukyu are also missing glyphs.

**Comment by davelab6** [comment]
Chrome on Windows 10 looks fine, but clearly there is more to investigate
here. But this is not urgent

**Comment by davelab6** [comment]
I am filing this as a question about the fonts; it seems to be a question about how to set up font fallback correctly. Maybe the font has issues (eg unicodes in the cmap table with no glyphs associated, I've seen that in some CJK fonts, which stops fallback from working correctly.)

**Comment by laerm0** [comment]
Yes, @davelab6, your assumption is essentially correct: Hannari is Kana-only^. So then the question really is how to substitute/set fallback for a more appropriate Kanji style for fonts like this. Could we have our embed code automatically put in the fallback for a font without Kanji? (And by "automatically," I mean somebody – i.e. me – goes through the Japanese fonts, notes which are missing Kanji, and then figures out if it makes the most sense with Gothic, Mincho, Kaku, Maru, etc?)

^ Glyphs says it has 174/181, missing vu-hira, voicedcomb-kana, semivoicedcomb-kana, va-kata, vi-kata, ve-kata, vo-kata

**Comment by davelab6** [comment]
Is it not possible to set up font fallback correctly so that the reasonable expectation is met, and the font really must have all that duplicate data inside it?
